### PR TITLE
addpkg: qalculate-qt

### PR DESCRIPTION
**Pull request by Ast-x64**

### CODE REVIEW

File: qalculate-qt/fix_arch.patch [hunk 1]
```diff
@@ -0,0 +1,22 @@
+diff --git a/.qmake.stash b/.qmake.stash
```
Comment (felixonmars) [inline qalculate-qt/fix_arch.patch:1]: Perhaps this file should not exist at all in the source tarball?

Comment (felixonmars) [inline qalculate-qt/fix_arch.patch:1]: Perhaps this file should not exist at all in the source tarball?

**Comment by moodyhunter** [comment]
Hello, @Ast-x64 why are you attempting to remove such a file in `prepare()` while this file doesn't exist at all from the release tarball?

**Comment by r-value** [comment]
Hi @moodyhunter , this file do exist when this PR was merged. It disappears now and we'll remove this patch later

**Comment by moodyhunter** [comment]
Thanks for your response

**Pull request merged**